tests/x11regressions/firefox/firefox_java.pm

```diff
@@ -53,13 +53,21 @@ sub java_testing {
 
 sub run() {
     my ($self) = @_;
-    $self->start_firefox;
-
-    assert_and_click('firefox-logo');
+    # Need to change owner of serial devide on opensuse
+    $self->start_firefox(check_var('DISTRI', 'opensuse'));
+    #Required only on sle, as open mozilla home page
```

Comment:
This is only remaining open point, verification run here: http://gershwin.arch.suse.de/tests/110#